CHANGELOG.md

```diff
@@ -17,10 +17,10 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### 📈 Features/Enhancements
 
-- [Multiple DataSource] Add support for SigV4 authentication ([#3058](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3058)). Backwards-compatible feature included in v2.6.0 release.
```

Comment:
Is this already elsewhere in the changelog?

Reply:
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-2.6.0.md

this item is in 2.6 release notes already

Reply:
Oh, yeah, now I remember - this is kind of a weird case. The reason I left a line in the unreleased section is that the change in `main` is distinct from the backwards-compatible equivalent we made in 2.6. (I could have just read my own comment 😆). I think it's fine to remove, but I'm not sure what's the most useful way to handle cases like that.